packages/badge/docs.mdx

```diff
@@ -6,36 +6,74 @@ Badges are non-interactive labels highlighting metadata such as status or catego
 
 <LiveComponent>
   {`
-<>
-  <div style={{ display: 'flex', flexWrap: 'wrap', gap: 16 }}>
-    <Badge theme="vault">New</Badge>
-    <Badge theme="consul">New</Badge>
-    <Badge theme="nomad">New</Badge>
-    <Badge theme="terraform">New</Badge>
-    <Badge theme="waypoint">New</Badge>
-    <Badge theme="boundary">New</Badge>
-    <Badge theme="packer">New</Badge>
-    <Badge theme="vagrant">New</Badge>
-    <Badge theme="neutral">New</Badge>
-    <Badge theme="black">New</Badge>
-    <Badge theme="white">New</Badge>
+  <div style={{ display: 'flex', gap: 16 }}>
+    <Badge variant="primary" theme="action" page="light" text="Badge" />
   </div>
-
-  <div style={{ display: 'flex', flexWrap: 'wrap', gap: 16, marginTop: 32 }}>
-    <Badge theme="vault" variant="secondary">New</Badge>
-    <Badge theme="consul" variant="secondary">New</Badge>
-    <Badge theme="nomad" variant="secondary">New</Badge>
-    <Badge theme="terraform" variant="secondary">New</Badge>
-    <Badge theme="waypoint" variant="secondary">New</Badge>
-    <Badge theme="boundary" variant="secondary">New</Badge>
-    <Badge theme="packer" variant="secondary">New</Badge>
-    <Badge theme="vagrant" variant="secondary">New</Badge>
-    <Badge theme="neutral" variant="secondary">New</Badge>
-    <Badge theme="black" variant="secondary">New</Badge>
-    <Badge theme="white" variant="secondary">New</Badge>
-  </div>
-</>
 `}
 </LiveComponent>
 
 <PropsTable props={componentProps} />
+
+## Examples
+
```

Comment:
This is pretty wild but it does render every possible variation of the `<Badge />`. Not sure how else we would be able to QA.